iroha/src/crypto.rs

```diff
@@ -98,7 +139,7 @@ pub struct Signature {
 impl Signature {
     /// Creates new `Signature` by signing payload via `private_key`.
     pub fn new(key_pair: KeyPair, payload: &[u8]) -> Result<Signature, String> {
-        let private_key = UrsaPrivateKey(key_pair.private_key.to_vec());
+        let private_key = UrsaPrivateKey(key_pair.private_key.inner.to_vec());
```

Comment:
Maybe implement `From<PrivateKey> for Vec<u8>`?

Reply:
It's only one place with it for now and we are planning to move to multihash and change display and serialization formats to hex, so I tried to avoid additional impls